Fixed a `ValidationError` that was being thrown when a user logged out.
